examples/go-samples/containerized/container/run.go

```diff
@@ -0,0 +1,45 @@
+package container
+
+import (
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/hashicorp/go-hclog"
+)
+
+// Run the container using Docker
```

Comment:
Expand on this perhaps, for example:

We assume (for now) that the container must return JSON and only pull back a single value from the returned JSON, based on the value of In.Key